src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -0,0 +1,70 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+public class Drivetrain extends SubsystemBase {
+
+  private CANSparkMax leftMotor = new CANSparkMax(0, MotorType.kBrushless);
+  private CANSparkMax rightMotor = new CANSparkMax(1, MotorType.kBrushless);
```

Comment:
Our robot will have 4 drive motors. Use a DifferentialDrive to convert from 2 inputs to 4 motors

https://first.wpi.edu/wpilib/allwpilib/docs/release/java/edu/wpi/first/wpilibj/drive/DifferentialDrive.html